### ISSUE TITLE
implementation with KL-loss

### ISSUE THREAD

**Issue by xiaowei-ui** [yihui-he/KL-Loss#21]
i want to know,do anyone have reproduced the code successfully? i want to apply the code into faster rcnn(tensorflow or pytorch),but i don't know how to do,looking forward to your recovery!

**Comment by ethanhe42** [comment]
Duplicate of #20

**Comment by LittleBoy7** [comment]
@zhangliliang  Do you incorporated successfully?

**Comment by luckyxianxian** [comment]
I would also like the pytorch version of the code very much and would appreciate it if the author could provide it
